api/global/internal/meter.go

```diff
@@ -264,29 +239,23 @@ func (bound *syncHandle) Unbind() {
 
 // Async delegation
 
-func (m *meter) newAsync(desc metric.Descriptor, constructor func(metric.Meter) (metric.AsyncImpl, error)) (metric.AsyncImpl, error) {
+func (m *meterImpl) NewAsyncInstrument(desc metric.Descriptor,
```

Comment:
Is the line-break inclusion intended here?

Reply:
It wasn't perfect. It was feeling long to me, so It's now written as:
```
func (m *meterImpl) NewAsyncInstrument(
	desc metric.Descriptor,
	callback func(func(metric.Number, []core.KeyValue)),
) (metric.AsyncImpl, error) {
```
I'm happy to go back to one line though if you think we should.